zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/Booter.java

```diff
@@ -78,20 +82,32 @@ static String resolveLocalRepoPath(String localRepoPath) {
     return Paths.get(home).resolve(localRepoPath).toAbsolutePath().toString();
   }
 
-  public static RemoteRepository newCentralRepository(Proxy proxy) {
-    String mvnRepo = System.getenv("ZEPPELIN_INTERPRETER_DEP_MVNREPO");
-    if (mvnRepo == null) {
-      mvnRepo = ZeppelinConfiguration.create().getString(
+  public static List<RemoteRepository> newCentralRepository(Proxy proxy) {
```

Comment:
rename it to `newCentralRepositories`

Reply:
I think we could add a test to check proper input/output. Could you please add it?

Reply:
> rename it to `newCentralRepositories`

thanks for review @zjffdu , I've renamed it.

Reply:
> I think we could add a test to check proper input/output. Could you please add it?

thanks for review @jongyoul, I would add test for it.

Reply:
> > I think we could add a test to check proper input/output. Could you please add it?
> 
> thanks for review @jongyoul, I would add test for it.

done